library/MedEx/API.php

```diff
@@ -2058,7 +2069,8 @@ public function login() {
         $this->curl->setData(array(
```

Comment:
Finally scrolled down past the giant word-pressy things on their site to find out how much it costs.  Something tells me that our outfit isn't going to be able to afford the $62,400 a year subscription fees for text messaging...